### PR TITLE
feat(assertions): rework method names

### PR DESCRIPTION
**Pull request by fdelbrayelle**
Fixes https://github.com/jhipster/jhipster-lite/issues/2484

### CODE REVIEW

**Review by DamnClin — COMMENTED** [review]
I don't think this is a good time to do this since some module migration are in progress, it will mean regressions (iit a breaking change). I think we should wait for the in progress migration by @DanielFran and @antarus to be merged

File: src/test/java/tech/jhipster/lite/module/infrastructure/secondary/JHipsterModulesAssertions.java [hunk 1]
```diff
@@ -86,23 +86,23 @@ private ModuleAsserter(JHipsterModule module) {
...
 
-    public ModuleAsserter createJavaSources(String... files) {
-      return createPrefixedFiles("src/main/java", files);
+    public ModuleAsserter hasJavaSources(String... files) {
+      return hasPrefixedFiles("src/main/java", files);
     }
 
-    public ModuleAsserter createJavaTests(String... files) {
-      return createPrefixedFiles("src/test/java", files);
+    public ModuleAsserter hasJavaTests(String... files) {
```
[the remark this answers is not in the document]
Comment (fdelbrayelle) [reply src/test/java/tech/jhipster/lite/module/infrastructure/secondary/JHipsterModulesAssertions.java]: Done: https://github.com/jhipster/jhipster-lite/pull/2539/commits/7937913a1c7f362741051da130987c184261802a

**Comment by pascalgrimaud** [comment]
I think it can be rebased or it could be done in another PR?

**Comment by fdelbrayelle** [comment]
More updated: https://github.com/jhipster/jhipster-lite/pull/3539

**Pull request closed without merging**